__tests__/targets/node/fetch.js

```diff
@@ -1,73 +1 @@
-module.exports = function (HTTPSnippet, fixtures) {
-  test('should respond with stringify when useObjectBody is enabled', function () {
```

Comment:
These tests are no longer needed since `useObjectBody: true` is now the default and only behavior for the `node-fetch` target.